examples/persistent-storage/efr32/README.md

```diff
@@ -155,3 +153,12 @@ combination with JLinkRTTClient as follows:
 -   In a second terminal, run the JLinkRTTClient to view logs:
 
           $ JLinkRTTClient
+
+## Memory settings
+
+While most of the RAM usage in CHIP is static, allowing easier debugging and
```

Comment:
I believe rendezvous objects as well as mDNS ones will use PlatformHeap. Are we considering PlatformHeap here as 'static' data since it uses a buffer?

We may want some GH issue/TODO element here to finalize info once CHIP is more fully-features (I expect the 5K number to change)

Reply:
One problem I encounter is that a lot of things inside CHIP are using a lot of different memory pool which cause the ram usage to be "bloated" unnecessarily. In the case that you mention it would be "static" data since it's inside a buffer and we can keep track of it at compiled time. But should it be ? what's the duration of the allocations ? what's the High-water mark on this buffer ? When does those allocations occurs ? That's a couple of questions we need to ask ourselves since most of the smallest part from us manufacturer are starting to be too small for CHIP RAM wise.